recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.1.7.1" %}
+{% set sha256 = "47699ba598570ec4eb2f852a2e7db268767a4d3b9042510259dd4492dfc7dee0" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    fn: py_gd-{{ version }}.tar.gz
+    url: https://github.com/NOAA-ORR-ERD/py_gd/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    skip: True  # [py3k or win]
+    number: 0
+    script:
+        - python -m pip install --no-deps ./
```

Comment:
This can just be put on one line with `script` and `-` dropped, but not a big deal.